public/app/plugins/datasource/prometheus/querybuilder/testUtils.ts

```diff
@@ -2,7 +2,7 @@ import { screen, getAllByRole } from '@testing-library/react';
 
 export function getLabelSelects(index = 0) {
   const labels = screen.getByText(/Labels/);
-  const selects = getAllByRole(labels.parentElement!, 'combobox');
+  const selects = getAllByRole(labels.parentElement!.parentElement!.parentElement!, 'combobox');
```

Comment:
Why did this change? Does not seem like this PR should change nesting in the select

Reply:
Because we are not using `EditorField` anymore and instead of that we are using `Field`.  We talked about it here: https://raintank-corp.slack.com/archives/C03045EE1EC/p1650368975000329. And both of this components have different html for `label`. In `Field` we are using `Label` component which has more nesting.